src/cms/views/dashboard/dashboard_view.py

```diff
@@ -67,18 +68,33 @@ def get(self, request, *args, **kwargs):
 
         # RSS FEED
         language_slug = translation.get_language()
-        feed = feedparser.parse(RSS_FEED_URLS[language_slug])
-        # select five most recent feeds
-        feed["entries"] = feed["entries"][:3]
-        # decode html entities like dash and split after line break
-        for entry in feed["entries"]:
-            entry["summary"] = html.unescape(entry["summary"]).split("\n")[0]
+        has_timed_out = False
+        feed = []
+        rss_feed_url = RSS_FEED_URLS[language_slug]
+
+        try:
+            requests.get(rss_feed_url, timeout=5.0)
+        except requests.ReadTimeout:
+            logger.debug(
+                "Rss-feed timed out. Continue loading site without news column."
+            )
+            has_timed_out = True
+
+        if not has_timed_out:
+            feed = feedparser.parse(rss_feed_url)
+            # select five most recent feeds
+            feed["entries"] = feed["entries"][:5]
+            # decode html entities like dash and split after line break
+            for entry in feed["entries"]:
+                entry["summary"] = html.unescape(entry["summary"]).split("\n")[0]
```

Comment:
Normally, in a try-catch block, you just continue the execution in the try-block as long as you want to rely on the "tried" command:
```suggestion
            # select five most recent feeds
            feed["entries"] = feed["entries"][:5]
            # decode html entities like dash and split after line break
            for entry in feed["entries"]:
                entry["summary"] = html.unescape(entry["summary"]).split("\n")[0]
        except requests.ReadTimeout:
            logger.debug(
                "Rss-feed timed out. Continue loading site without news column."
            )
            has_timed_out = True
```